specification.swagger.yaml

```diff
@@ -965,57 +694,762 @@ paths:
         - name: id
           in: path
           required: true
-          type: integer
-          description: Internal id of street. Notice, not persistent. Should only be used when resolving street ids from address resources.
+          description: Internal id of street. Notice, not persistent. Should only be used
+            when resolving street ids from address resources.
+          schema:
+            type: integer
+          example: 2
       responses:
-        200:
+        "200":
           description: A single street resource
-          schema:
-            $ref: '#/definitions/street'
-
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/street"
   /address/:
     get:
       operationId: Retrieve a list of addresses
       summary: Retrieve a list of addresses
       parameters:
         - name: lat
           in: query
-          type: string
           description: The latitude coordinate to be used in reverse geocoding
+          schema:
+            type: number
+          example: 20.20
         - name: lon
           in: query
-          type: string
           description: The longitude coordinate to be used in reverse geocoding
+          schema:
+            type: number
+          example: 20.20
         - name: language
           in: query
-          type: string
-          description: The two-character ISO 639-1 language code to be used in the street names of the addresses. If missing, the street name parameter will use the default language Finnish.
+          description: The two-character ISO 639-1 language code to be used in the street
+            names of the addresses. If missing, the street name parameter will
+            use the default language Finnish.
+          schema:
+            type: string
+          example: fi
         - name: municipality
           in: query
-          type: string
-          description: A municipality id. The returned addresses will be located within the municipalitie\'s boundaries. The municipality id is either a municipality name in Finnish, or a full OCD ID of the form ocd-division/country\:fi/kunta\:helsinki (URL encoded) See the [OpenCivicData Site](http://opencivicdata.readthedocs.io/en/latest/ocdids.html) for more information
+          description: A municipality id. The returned addresses will be located within the
+            municipalitie\'s boundaries. The municipality id is either a
+            municipality name in Finnish, or a full OCD ID of the form
+            ocd-division/country\:fi/kunta\:helsinki (URL encoded) See the
+            [OpenCivicData
+            Site](http://opencivicdata.readthedocs.io/en/latest/ocdids.html) for
+            more information
+          schema:
+            type: string
+          example: espoo,vantaa
```

Comment:
This does not work either. I guess this does not accept lists?

Reply:
It does and is actively used in the Servicemap UI:

    https://api.hel.fi/servicemap/v2/unit/?service_node=1061&municipality=helsinki%2Cespoo&only=root_service_nodes%2Cservices%2Clocation%2Cname%2Cstreet_address%2Ccontract_type&include=service_nodes%2Cservices%2Caccessibility_properties&geometry=true&page=1&page_size=1000

Reply:
Sorry, I didn't notice this was about the address endpoint.

Reply:
Already talked about this with Timo. This is the address endpoint:
https://api.hel.fi/servicemap/v2/address/?municipality=espoo,vantaa

Reply:
Fixed